src/pipeline.ts

```diff
@@ -777,6 +895,10 @@ export class GitHubWorkflow extends PipelineBase {
 
     return deps.map(x => x.uniqueId);
   }
+
+  private renderJobSettingParameters() {
+    return this.jobSettings;
+  }
```

Comment:
okay, lets keep it, but probably not necessary

Reply:
:+1: